test/IntegrationTests/ModuleTests.Default.NetCore.verified.txt

```diff
@@ -5,10 +5,8 @@
   OpenTelemetry.AutoInstrumentation.Loader,
   OpenTelemetry.AutoInstrumentation.StartupHook,
   OpenTelemetry.Exporter.Console,
```

Comment:
Starting the review here - I may see why later, but, do you know why `Console` and `Prometheus` are still being loaded?

Reply:
> This PR fixes only tracing related parts

Yes, these are coming via Metrics.